### DISCUSSION TITLE
MAUI Maps issue

### DISCUSSION THREAD

**Discussion by kevinvenclovas** [VladislavAntonyuk/MauiSamples#420] [Q&A] [answered]
Hi,

love your MAUI Projects.

Maybe you can help me with my MAUI Maps problem.

I have multiple map elements in my MAUI Map project and i'm very unhappy about the overlapped area.

I can't find any simple solution out there. Maybe you have an idea how i can merge the two circle so the overlapped area did not looks darker?

![image](https://github.com/VladislavAntonyuk/MauiSamples/assets/7681603/5732b4e0-37a1-420d-b5b6-cfd60571bf86)

```
var circle = new Circle
{
    Center = new Location(48.9624, 10.1295), // Koordinaten für das Zentrum des Kreises (z.B. Berlin)
    Radius = new Distance(100), // Radius in Metern
    StrokeColor = Color.FromArgb("50FF0000"), // Farbe des Kreisrandes
    StrokeWidth = 1, // Dicke des Kreisrandes
    FillColor = Color.FromArgb("50FF0000"),// Füllfarbe des Kreises (Rot mit etwas Transparenz)
};

var circle1 = new Circle
{
    Center = new Location(48.9619, 10.1315), // Koordinaten für das Zentrum des Kreises (z.B. Berlin)
    Radius = new Distance(100), // Radius in Metern
    StrokeColor = Color.FromArgb("50FF0000"), // Farbe des Kreisrandes
    StrokeWidth = 1, // Dicke des Kreisrandes
    FillColor = Color.FromArgb("50FF0000") // Füllfarbe des Kreises (Rot mit etwas Transparenz),
};

map.MapElements.Add(circle);
map.MapElements.Add(circle1);
```

**Comment by VladislavAntonyuk** [comment, marked as answer]
This is a js example, but probably a good start point: https://stackoverflow.com/questions/46726086/prevent-overlapping-colours-becoming-darker-in-google-maps

This is for .NET MAUI: https://www.andreasnesheim.no/creating-outlined-map-polygons-in-net-maui/
